### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by kobylianskiy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,44 +0,0 @@
...
-        </div>
-        <div class="form-group">
-            <label for="assigned"> Assigned To </label>
-            <input type="text" class="form-control" id="assigned" ng-model="assignee">
-        </div>
-        <button type="submit" class="btn btn-primary" ng-click="addTask()"> Create Task</button>
-    </form>
-
-    <ul class="list-group">
-        <li ng-style="task.getIsFinished() &&  {'text-decoration': 'line-through'}" class="list-group-item" ng-repeat="task in tasks() | taskOrderBy track by $index" ng-mouseover="task.show = true" ng-mouseleave="task.show = false">
```
Comment (natalia-zolochevska) [inline index.html]: discuss naming of boolean var

File: js/app.js [hunk 2]
```diff
@@ -1,4 +1,16 @@
-angular.module('ToDoApp', [])
-    .config(function ($locationProvider) {
-        $locationProvider.html5Mode(true);
+angular.module('ToDoApp', ['ui.router'])
+    .config(function ($stateProvider) {
+        $stateProvider
+            .state('tasks', {
+                url: '/tasks',
+                templateUrl: 'views/list.html',
+                controller: 'TaskListController'
```
Comment (natalia-zolochevska) [inline js/app.js:7]: controller name starts with lower case

File: views/edit.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<form name="editTask" class="container" ng-submit="updateTask()">
+    <h2>Task</h2>
+
+    <div class="form-group">
+        <label for="assignee">Assignee</label>
+        <input name="assignee" type="text" class="form-control" id="assignee" ng-model="assignee" td-description>
+        <span class="error" ng-show="editTask.assignee.$dirty && editTask.assignee.$invalid">The value is not a valid!</span>
```
Comment (natalia-zolochevska) [inline views/edit.html]: prefer css

File: views/list.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+<ul class="list-group">
+    <li ng-class="{ cross: task.isFinished }" class="list-group-item"
+        ng-repeat="task in tasks() | taskOrderBy" ng-mouseover="show = true" ng-mouseleave="show = false">
+        <span ng-click="editPage(task.id)">
+            {{ task.description }}
+        </span>
+
+        <div class="pull-right" ng-show="show">
+            <button type="button" class="btn btn-danger" ng-click="removeTask(task.id)"> Remove</button>
+            <button type="button" class="btn btn-success" ng-click="changeState(task.isFinished, task.id)">
```
Comment (natalia-zolochevska) [inline views/list.html]: discuss naming of boolean attrbiutes

File: views/list.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        <input name="assignee" type="text" class="form-control" id="assigned" ng-model="assignee" td-description>
+        <span class="error" ng-show="createTask.assignee.$dirty && createTask.assignee.$invalid">The value is not a valid!</span>
+    </div>
+    <button type="submit" class="btn btn-primary" ng-click="addTask()" ng-disabled="createTask.$invalid"> Create Task</button>
+</form>
+
+<ul class="list-group">
+    <li ng-class="{ cross: task.isFinished }" class="list-group-item"
+        ng-repeat="task in tasks() | taskOrderBy" ng-mouseover="show = true" ng-mouseleave="show = false">
+        <span ng-click="editPage(task.id)">
```
Comment (natalia-zolochevska) [inline views/list.html]: gotoEditPage or editTask

**Comment by sergiyenko** [comment]
LGTM

**Comment by digitalmaxing** [comment]
LGTM

**Pull request merged**